src/mapping/Mapping.hpp

```diff
@@ -184,6 +257,12 @@ class Mapping {
   mesh::PtrMesh _output;
 
   int _dimensions;
+
+  /// The type of the mapping
+  Type _mappingType;
```

Comment:
Note that "type of mapping" can mean anything, as we have several categorizations already (e.g., consistent-conservative, with the geometric-multiscale also adding collect-spread, axial-radial, etc). Not sure if there are better terms, though.

Reply:
Naming is tricky here. I really couldn't come up with something better.

Reply:
After a discussion, we came up with `InitialGuessRequirement` `Required` and `None`.